src/SurveyDesign.jl

```diff
@@ -30,53 +29,93 @@ The population size is equal to the sample size unless `popsize` is explicitly p
 """
 struct SimpleRandomSample <: AbstractSurveyDesign
     data::DataFrame
-    sampsize::UInt
-    popsize::Union{UInt,Nothing}
+    sampsize::Union{Nothing,Unsigned}
```

Comment:
Why is this change necessary?

Reply:
Unsigned is the `supertype` for UInt, more correct